docker/studyu/Dockerfile

```
@@ -1,41 +1,56 @@
-FROM ghcr.io/cirruslabs/flutter:stable as builder
+FROM debian:latest as builder
+
+RUN apt-get update
+RUN apt-get install -y bash curl file git libglu1-mesa unzip which xz-utils zip
+RUN apt-get clean
+
+RUN useradd -ms /bin/bash builder
+USER builder
+
+ENV HOME=/home/builder
+ENV PATH="${PATH}:${HOME}/flutter/bin:${HOME}/.pub-cache/bin"
+
+RUN git clone -b master https://github.com/flutter/flutter.git ~/flutter
+
+RUN flutter channel stable
+RUN flutter config --enable-web
+RUN flutter upgrade
+RUN flutter doctor -v
 
 ARG FLUTTER_APP_FOLDER
 
-ENV PATH ${PATH}:/root/.pub-cache/bin/
+# Create workdir and copy files
+RUN mkdir ~/app/
+COPY --chown=builder:builder . ~/app/
+WORKDIR ~/app/
 
-# Create workdir
-WORKDIR /src/
+# Disable parallel pub get
+RUN sed -i 's/runPubGetInParallel: true/runPubGetInParallel: false/' melos.yaml
 
 # Install melos
 RUN dart pub global activate melos
 
-# Copy repository files
-COPY ./ ./
-
 # Clean repository without failing
-RUN melos clean
+RUN melos --verbose clean
```

All --verbose should be removed before merge

---

Thanks for the hint.

```suggestion
RUN melos clean
```